doc/glossary/index.rst

```diff
@@ -31,3 +31,374 @@ Glossary
+        standalone unix systems. Web services often use SQL databases as
+        user store.
+
+        Today with many more online cloud services SCIM is also an uprising
+        protocol to access userstores.
+
+        privacyIDEA already comes with :ref:`useridresolvers` to talk to all these
+        user stores:
+
+        * :ref:`flatfile_resolver`
+        * :ref:`ldap_resolver`
+        * :ref:`sql_resolver`
+        * :ref:`scim_resolver`
+        * :ref:`http_resolver`
+
+   Resolver(Machine)
+        Machine Resolvers are used to find machines in directories like LDAP,
+        Active Directory, puppet, salt, or the /etc/hosts file.
+
+        The idea is for users to be able to authenticate on those client machines.
+        Not in all cases an online authentication request is possible,
+        so that authentication items can be passed to those client machines.
+
+        In addition you need to define, which application on the client machine
+        the user should authenticate to.
+        Different application require different authentication items.
+
+        Therefore privacyIDEA can define application types.
+        At the moment privacyIDEA knows the application
+        ``luks``, ``offline`` and ``ssh``.
+
+   Tokeninfo
+        The table “tokeninfo” is used to store additional, long information that is
+        specific to the :ref:`tokentypes`. E.g. the tokentype “TOTP” has additional entries
+        in the tokeninfo table for “timeStep” and “timeWindow”, which are stored in the
+        column “Key” and “Value”.
+
+        The ``tokeninfo`` is reference by the foreign key to the “token” table.
+
+        Token info can be viewed and partially edited in the GUI. In addition, the
+        Token-Janitor can be used to output token info, filter for tokens that have
+        specific tokeninfo and set user-defined tokeninfos.
+
+   Token
+        All token information is stored in an SQL database, while you may choose, which database
+        you want to use. privacyIDEA uses SQLAlchemy to map the database to internal objects.
+        Thus you may choose to run privacyIDEA with SQLite, MySQL, PostgreSQL, Oracle,
+        DB2 or other database.
+
+        PrivacyIDEA supports a great variety of different token types.
+        They each have different requirements concerning configuration and how
+        the authentication works. This chapter explains the authentication modes, lists the
+        supported hardware and software tokens and explains how the token types can be used
+        with privacyIDEA.
+        Tools which facilitate and automate token enrollment are found in :ref:`enrollment_tools`.
+
+        * :ref:`authentication_modes`
+        * :ref:`supported_tokens`
+        * :ref:`tokentypes`
+
+   Audit
+        The systems provides a sophisticated audit log, that can be viewed in the WebUI.
+
+        The Audit log lists all events the server registers.
```

Comment:
registers?

Reply:
done in 77a9cf862391a2715de0ba765c304263757524b9.